packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -6,16 +6,49 @@ import {
 } from 'grommet'
 import PropTypes from 'prop-types'
 import { useCallback, useEffect, useRef } from 'react';
+import styled from 'styled-components'
 import { withThemeContext } from '@zooniverse/react-components'
 
 import theme from './theme'
 
 export const THUMBNAIL_ASPECT_RATIO = 1.25
 
+const StyledBox = styled(Box)`
```

Comment:
[idea?] Wait, a thought occurs: we _could_ list out and programmatically test some of those possibilities with Storybook and React Testing Library. I mean, it wouldn't catch everything, because there's a lot of eyeballin' needed to see if the UI "looks good and readable", but it may still help.

Reply:
All modern browsers now support auto-flow columns with CSS grids, so we should be able to remove code that calculates `columnCount`. That code was originally written before grids came along, so there's some work here that the browser can now do for us. That's something that's worth looking at, though not a blocker for this PR.

Reply:
Definitely agree the media and column count combinations and how they determine choice button styling could use some additional explanation/testing/something!

I don't think RTL knows about media width? Would that be related to jsdom? I'm not sure how to write RTL tests conditional to media queries.

I attempted to create additional stories, but realized the built-in storybook responsive tool I think does what I was trying to do with the existing stories well. Though it looks a little strange without a subject image. I tried adding the DefaultLayout to the Choices stories to include a subject image and the related grid styling, but that proved to be a lot more challenging than anticipated (requires mock store that doesn't work well with MockTask used in task stories, I think). 

I added some additional comments to the ChoiceButton spec. It's not the ideal solution, but I think good enough for now, though open to suggestion!

Reply:
Noting [the Choices stories](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/tasks-survey-chooser-choices--less-thirty-more-twenty) (link does not include this PR's changes) include choices with 24 options (3 columns), 6 options (2 columns), and 4 options (1 column).